cpu/esp_common/Kconfig

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2020 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config CPU_ARCH_XTENSA
+    bool
+    select HAS_ARCH_32BIT
+
```

Comment:
```suggestion
select HAS_ARCH_ESP

```
see PR #14375

Reply:
Why using a feature for that when `CPU_ARCH` could be checked?